civihr_default_theme/scss/components/_button.scss

```diff
@@ -2,7 +2,11 @@
   border: transparent;
   padding: 0 10px;
 
-  & + .dropdown-menu > li > a {
+  + .dropdown-menu a {
```

Comment:
why this change? There is no mention in the PR

Reply:
To satisfy the linter.